bindings/c/test/unit/unit_tests.cpp

```diff
@@ -2709,6 +2708,42 @@ TEST_CASE("Blob Granule Functions") {
 		tr.reset();
 		break;
 	}
+
+	// do a purge + wait at that version to purge everything before originalReadVersion
+
+	fdb::Database database;
+	fdb::KeyFuture purgeKeyFuture = database.purge_blob_granules(db, key("bg"), key("bh"), originalReadVersion, false);
```

Comment:
Don't use an object for calling static methods. It just creates a confusion

Reply:
Oh I forgot the database methods are static, fixing